test/unit/Clustering/GMUSimpleClusterAlgorithmTest.swift

```diff
@@ -0,0 +1,59 @@
+/* Copyright (c) 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import XCTest
+@testable import GoogleMapsUtils
+
+class GMUSimpleClusterAlgorithmTest: GMUClusterAlgorithmTest {
```

Comment:
I'd prefer for this not to inherit from another test case since that pattern is quite uncommon—also best to minimize inheritance in general.

Reply:
I agree, but in the project, I have seen `@interface GMUNonHierarchicalDistanceBasedAlgorithmTest : GMUClusterAlgorithmTest` this. and Why I found it useful because I can reuse this method `self.simpleClusterItems()` without rewriting it. Let me know what's your call on this.

Reply:
I see. I'd prefer for `GMUSimpleClusterAlgorithmTest` to not inherit from `GMUClusterAlgorithmTest`. If you want to reuse `simpleClusterItems` it would be better to create a separate helper class for that.

Reply:
@arriolac as discussed, we need to redesign as `"GMUClusterAlgorithmTest.h"` is called in multiple places.